requests/models.py

```diff
@@ -577,6 +577,10 @@ def prepare_hooks(self, hooks):
         for event in hooks:
             self.register_hook(event, hooks[event])
 
+    def send(self, session):
```

Comment:
more intuitive. i dabbled about it alot when i was writing it originally, and i opted out, decided that i could always add it back in later — i think it's time for it to go in. it's just a nice shortcut.